lib/Value/Matrix.pm

```diff
@@ -6,128 +6,137 @@
 
 =head1 Value::Matrix class
 
-
 References:
 
-MathObject Matrix methods: L<http://webwork.maa.org/wiki/Matrix_(MathObject_Class)>
-MathObject Contexts: L<http://webwork.maa.org/wiki/Common_Contexts>
-CPAN RealMatrix docs: L<http://search.cpan.org/~leto/Math-MatrixReal-2.09/lib/Math/MatrixReal.pm>
+=over
 
-Allowing Matrices in Fractions:
-L<http://webwork.maa.org/moodle/mod/forum/discuss.php?d=2978>
+=item MathObject Matrix methods: L<http://webwork.maa.org/wiki/Matrix_(MathObject_Class)>
 
-     Context()->parens->set("[" => {formMatrix => 1});
+=item MathObject Contexts: L<http://webwork.maa.org/wiki/Common_Contexts>
 
-Files interacting with Matrices:
+=item CPAN RealMatrix docs: L<http://search.cpan.org/~leto/Math-MatrixReal-2.09/lib/Math/MatrixReal.pm>
 
-L<MatrixReal1>
+=back
 
-L<MatrixReduce.pl>
+For allowing Matrices in Fractions, see L<http://webwork.maa.org/moodle/mod/forum/discuss.php?d=2978>
 
-L<Matrix>
+    Context()->parens->set("[" => {formMatrix => 1});
 
-L<MatrixCheckers.pl> -- checking whether vectors form a basis
+=head2 Files interacting with Matrices:
```

Comment:
It took me a sec to understand this phrase (which I know was already there before this PR). Not sure if the suggestion below is any better but maybe. 

```suggestion
=head2 Matrix-related Files:
```